python/caffe/_caffe.cpp

```diff
@@ -237,7 +260,12 @@ BOOST_PYTHON_MODULE(_caffe) {
 
   bp::class_<Net<Dtype>, shared_ptr<Net<Dtype> >, boost::noncopyable >("Net",
     bp::no_init)
-    .def("__init__", bp::make_constructor(&Net_Init))
+    // Constructor
+    .def("__init__", bp::make_constructor(&Net_Init,
+          bp::default_call_policies(), (bp::arg("network_file"), "phase",
+            bp::arg("level")=0, bp::arg("stages")=bp::object(),
+            bp::arg("weights_file")=bp::object())))
```

Comment:
How about `weights` instead of `weights_file`? Including `_file` is more explicit but I like short arg names and every example will be a path to a `.caffemodel`.